components/@ui/Select.jsx

```diff
@@ -1,5 +1,5 @@
 import { useContext, useState } from 'react';
-import DarkModeContext from '../../context/darkMode';
+import themeContext from '../../context/darkMode';
```

Comment:
I find it a bit confusing to import from `context/darkMode`. Might perhaps better express the intention to rename to `context/defaultTheme`?